inspirehep/modules/workflows/workflows/article.py

```diff
@@ -279,9 +288,29 @@
     ),
     IF_ELSE(
         is_marked('is-update'),
-        [
-            mark('approved', True)
-        ],
+        IF_ELSE(
+            is_merge_on_update_enabled,
```

Comment:
same as before for feature flags style, I would put this check in `merge_articles`